Spigot-API-Patches/0231-Damage-with-any-cause.patch

```diff
@@ -0,0 +1,66 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NeumimTo <chleba48@gmail.com>
+Date: Wed, 28 Oct 2020 18:42:07 +0100
+Subject: [PATCH] Damage with any cause
+
+
```

Comment:
This is licensed under GPLv3. Add a note to license it under other licences, or add your name to the current licence file to use MIT.

Reply:
Added myself to lic. file